.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Your team will decide whether remove this or not. Personally I would remove this.

Reply:
Removed.